index.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
           * Always delay requires, otherwise your plugin will cause trouble
           * with db-migrates performance and generates issues to your users.
           */
-        require('typescript').register();
+        var typescriptModule = require('typescript');
+        if (typescriptModule && typescriptModule.register) {
```

Comment:
In ES6 there is no more `register` function, so the plugin crashes even if the module is installed.
My update fixes this problem.